trinity/chains/__init__.py

```diff
@@ -82,30 +85,32 @@ def is_database_initialized(chaindb: AsyncChainDB) -> bool:
 
 
 def initialize_data_dir(chain_config: ChainConfig) -> None:
-    if is_under_xdg_trinity_root(chain_config.data_dir):
+    if (not os.path.exists(chain_config.data_dir) and
+            is_under_xdg_trinity_root(chain_config.data_dir)):
+
         chain_config.data_dir.mkdir(parents=True, exist_ok=True)
     elif not os.path.exists(chain_config.data_dir):
         # we don't lazily create the base dir for non-default base directories.
-        raise ValueError(
+        raise AmbigiousFileSystem(
             "The base chain directory provided does not exist: `{0}`".format(
                 chain_config.data_dir,
             )
         )
 
     # Logfile
-    if is_under_xdg_trinity_root(chain_config.logfile_path):
+    if (not os.path.exists(chain_config.logfile_path) and
```

Comment:
the `Path.exists()` method works here too, we can drop some of the `os.path` calls now that we're using `Path` objects most places.